<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>buySouvenir</class>
 <widget class="QMainWindow" name="buySouvenir">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Purchase Souvenir</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_receipt">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>651</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>Receipt of Purchase</string>
    </property>
    <widget class="QTextBrowser" name="textBrowser_cart_itemName_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>100</y>
       <width>161</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_cart_itemPrice_2">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>100</y>
       <width>91</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_cart_itemQty_2">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>100</y>
       <width>61</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_cart_stadiumName_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>100</y>
       <width>161</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Souvenir Item</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>80</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Purchased From</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>80</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Quantity</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>80</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Price</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_grandTotal">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>330</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>340</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 18pt &quot;Fixedsys&quot;;</string>
     </property>
     <property name="text">
      <string>Grand Total</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>20</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>461</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 18pt &quot;Fixedsys&quot;;</string>
     </property>
     <property name="text">
      <string>Thank you for your purchase!</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>textBrowser_cart_itemName_2</zorder>
    <zorder>textBrowser_cart_itemPrice_2</zorder>
    <zorder>textBrowser_cart_itemQty_2</zorder>
    <zorder>textBrowser_cart_stadiumName_2</zorder>
    <zorder>label</zorder>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
    <zorder>label_4</zorder>
    <zorder>lcdNumber_grandTotal</zorder>
    <zorder>label_5</zorder>
    <zorder>pushButton</zorder>
    <zorder>groupBox_main</zorder>
    <zorder>groupBox_main</zorder>
    <zorder>label_8</zorder>
    <zorder>label_9</zorder>
   </widget>
   <widget class="QGroupBox" name="groupBox_main">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>831</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>Make Purchase</string>
    </property>
    <widget class="QLCDNumber" name="lcdNumber_itemxqty">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>170</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="smallDecimalPoint">
      <bool>true</bool>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Filled</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_resetCart">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset Cart</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_confirmPurchase">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>251</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qconicalgradient(cx:0, cy:0, angle:135, stop:0 rgba(21, 61, 158, 69), stop:0.375 rgba(255, 255, 0, 69), stop:0.423533 rgba(251, 255, 0, 145), stop:0.45 rgba(247, 255, 0, 208), stop:0.477581 rgba(255, 244, 71, 130), stop:0.518717 rgba(255, 218, 71, 130), stop:0.55 rgba(255, 255, 0, 255), stop:0.57754 rgba(255, 203, 0, 130), stop:0.625 rgba(255, 255, 0, 69), stop:1 rgba(255, 255, 0, 69));
font: 14pt &quot;Comic Sans MS&quot;;</string>
     </property>
     <property name="text">
      <string>Confirm Purchase</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_cartTotal">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>320</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_cart_itemName">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>0</y>
       <width>161</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_qty_input">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_addToCart">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Add To Cart</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_cart_itemPrice">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>0</y>
       <width>91</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_cartQty">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>320</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_cart_itemQty">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>0</y>
       <width>61</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_cart_stadiumName">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>0</y>
       <width>161</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_stadiumName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_souvenir">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>300</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Total Qty</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>300</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Cart Total</string>
     </property>
    </widget>
    <zorder>lcdNumber_itemxqty</zorder>
    <zorder>pushButton_resetCart</zorder>
    <zorder>pushButton_confirmPurchase</zorder>
    <zorder>lcdNumber_cartTotal</zorder>
    <zorder>textBrowser_cart_itemName</zorder>
    <zorder>spinBox_qty_input</zorder>
    <zorder>pushButton_addToCart</zorder>
    <zorder>textBrowser_cart_itemPrice</zorder>
    <zorder>lcdNumber_cartQty</zorder>
    <zorder>textBrowser_cart_itemQty</zorder>
    <zorder>textBrowser_cart_stadiumName</zorder>
    <zorder>comboBox_stadiumName</zorder>
    <zorder>comboBox_souvenir</zorder>
    <zorder>label_6</zorder>
    <zorder>label_7</zorder>
    <zorder>groupBox_receipt</zorder>
    <zorder>groupBox_receipt</zorder>
   </widget>
   <zorder>groupBox_main</zorder>
   <zorder>groupBox_receipt</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>851</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
